lib/util/config-file.test.js

```diff
@@ -0,0 +1,22 @@
+import test from 'ava';
+import configFile from './config-file.js';
+
+const absentFile = './abc';
+const expected = 'Configuration file not found: ' + absentFile;
+
+test('Reading an absent configuration file throws a helpful error', t => {
+  t.is(
+    t.throws(() => {
+      configFile.readConfig(absentFile);
+    }).message,
+    expected
+  );
```

Comment:
You can simplify this like the following:

```js
t.throws(() => {
  configFile.readConfig(absentFile);
}, expected);
```
because `t.throws` already takes an optional parameter allowing to verify the message of the error. See https://github.com/avajs/ava#throwsfunctionpromise-error-message